dotCMS/src/integration-test/java/com/dotcms/publisher/bundle/business/BundleFactoryTest.java

```diff
@@ -0,0 +1,215 @@
+package com.dotcms.publisher.bundle.business;
+
+import com.dotcms.datagen.TestDataUtils;
+import com.dotcms.datagen.UserDataGen;
+import com.dotcms.publisher.bundle.bean.Bundle;
+import com.dotcms.publisher.business.DotPublisherException;
+import com.dotcms.publisher.business.PublisherAPI;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.FactoryLocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.DateUtil;
+import com.dotmarketing.util.UUIDGenerator;
+import com.liferay.portal.model.User;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+public class BundleFactoryTest {
+
+    private static BundleFactory bundleFactory;
+    private static User adminUser;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)